website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -0,0 +1,683 @@
+version and other values. Looking at the original `copy_to_glacier`, `s3.copy`
+is used to copy data to the ORCA archive. Based on initial research located
+[here](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.copy),
+the `s3.copy` command does not return any information. See the test performed
+below. Because of this, we will likely need to perform a listing on the object to
+get additional information like the version as seen in the sample seen below.
+Some things to consider when looking at the options
+for getting the additional data include speed (how fast can I get the data), cost
+(what is the cost impact of additional gets on the  s3 bucket), and complexity
+(where is the best place to put these additional gets, is it overly complex)
+
+#### Boto3 Copy Test
+
+```python
+import boto3
+
+# Create the Client
+s3_client = boto3.resource('s3')
+
+# Get a source and destination to copy
+copy_source = {"Bucket": "orca-sandbox-s3-provider", "Key": "MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065111.hdf"}
+copy_destination = "orca-sandbox-glacier"
+
+# perform the copy command
+copy_command = s3_client.meta.client.copy(copy_source, copy_destination, copy_source["Key"])
+
+# Return results
+print(copy_command)
+None
+```
+
+
+#### Boto3 Get Additional File Information
+
+This would occur after an `s3.copy` and assumes that the copy is synchronous and
+that the file is not be ingested in parallel in a separate ingest pipeline that
+would lead to a collision.
+
+```python
+import boto3
+
+# Create the Client
+s3_client = boto3.resource('s3')
+
+# Get a source and destination to copy
+copy_source = {"Bucket": "orca-sandbox-s3-provider", "Key": "MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065111.hdf"}
+copy_destination = "orca-sandbox-glacier"
+
+# Get the versioning information
+file_versions = s3_client.meta.client.list_object_versions(Bucket=copy_destination, Prefix=copy_source["Key"] )
+
+for version in file_versions["Versions"]:
+    if version["IsLatest"]:
+        print(version)
+
+
+# Return results
+{
+    'ETag': '"81f4b6c158d25f1fe916ea52e99d1700"',
+    'Size': 6,
+    'StorageClass': 'STANDARD',
+    'Key': 'MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065111.hdf',
+    'VersionId': 'null',
+    'IsLatest': True,
+    'LastModified': datetime.datetime(2021, 5, 18, 16, 25, 58, tzinfo=tzutc()),
+    'Owner': {
+        'DisplayName': 'gsfc-esdis-edc-lpdaac-sandbox-7343-root',
+        'ID': '4d102362d8dc848404655e5418ff76cd342418b8d2ea3a04aa5db133bd6ac1db'
+    }
+}
+```
+
+Note that the data we would likely want are ETag, VersionId, Size, and LastModified.
+In the example above, the file I used was not part of a versioned S3 bucket thus
+the `null` value for VersionId.
+
+
+#### Boto3 Use Multipart Copy
+
+In addition to the items above, I also looked at multipart copy which would
+return the information natively but, there is a 5Mb minimum size limit when
```

Comment:
return the information natively, but there is a 5Mb minimum size limit when

Reply:
fixed.